ivy_tests/test_ivy/test_functional/test_core/test_creation.py

```diff
@@ -23,12 +24,13 @@
         max_dim_size=10,
         shared_dtype=True,
     ),
-    as_variable=st.booleans(),
```

Comment:
we don't want to remove the arguments which are already here, particularly because many of them use `helpers.array_bools` to generate independent flags for each input array. The current approach would replace all of these with `st.booleans` which we do not want. We should receive the existing values as input to `handle_cmd_line_args`, and return the original if nothing has been set at the command line.

Reply:
Should we keep only the one's which use array bools or should we keep everything?